app/components/pages/Account/index.js

```diff
@@ -161,15 +172,20 @@ class Account extends React.Component {
                 <div className="pure-control-group selection-list">
                   <label>{t("form.career_ids.label")}</label>
```

Comment:
`htmlFor`?

Reply:
Doesn't really make sense here, as we have a list of inputs, so clicking the label should not change any of them